examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -63,6 +82,11 @@ CHIP_ERROR PairingCommand::RunInternal(NodeId remoteId)
 {
     CHIP_ERROR err = CHIP_NO_ERROR;
 
+#if CHIP_DEVICE_LAYER_TARGET_LINUX && CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+    struct sigaction sa = {};
```

Comment:
how about having this in the chip-tool main? That would make it clear what happens when the user stops things.

How does interact with https://github.com/project-chip/connectedhomeip/blob/master/examples/platform/linux/AppMain.cpp#L841? Is chip-tool using a different main? Or is this overriding that one (it should not I believe).

Reply:
AppMain.cpp is not used in chip-tool.